hack/make.sh

```diff
@@ -118,8 +118,10 @@ fi
 
 if [ -z "$DOCKER_CLIENTONLY" ]; then
 	DOCKER_BUILDTAGS+=" daemon"
-	if pkg-config libsystemd-journal 2> /dev/null ; then
+	if pkg-config 'libsystemd >= 209' 2> /dev/null ; then
```

Comment:
Aww, I was just being hopeful -- if it doesn't work from the cli, it likely won't work from Go either.  Thanks for indulging me. :innocent: